pt-BR/mail/chrome/messenger/folderpane.dtd

```diff
@@ -34,10 +34,10 @@
  the terms of any one of the MPL, the GPL or the LGPL.
 
  ***** END LICENSE BLOCK ***** -->
-<!ENTITY nameColumn.label   "Nome"> 
-<!ENTITY unreadColumn.label "Não lidas"> 
-<!ENTITY totalColumn.label "Total"> 
-<!ENTITY folderSizeColumn.label "Tamanho"> 
-<!ENTITY accounts.label "Accounts">
+<!ENTITY nameColumn.label   "Nome">
+<!ENTITY unreadColumn.label "Não lidas">
```

Comment:
I disagree. With the new Ortographic Agreement, there's no hifen after "não" anymore, no matter what's the next word.